src/containers/home/home.component.jsx

```diff
@@ -1,49 +1,54 @@
-import React from "react";
-
+import React, { useEffect, useState } from "react";
 import { auth } from "database";
-
-import { fetchPosts } from "database/posts";
-import { addPost } from "database/addPost";
-
 import { useSelector } from "react-redux";
+import CreatePost from "containers/create-post/create-post.component";
+import Post from "containers/post/post.component";
```

Comment:
```suggestion
import Post from "containers/post";
```

Reply:
same with above, nothing in the index.js file